CONTRIBUTION_GUIDE.md

```diff
@@ -0,0 +1,69 @@
+### Choosing Stories/tickets.
+
+When deciding on an issue to work on, look for the  `Help Wanted` or `Good First Issue` tags.
+
+### Branch Naming Conventions
+
+Kindly follow these conventions when creating branches and pull requests.
+```
```

Comment:
I would move this to a separated topic, and move the actual convention under this.
Maybe this part could be under the title 
How to create a feature branch

Reply:
@joaopapereira Fixed.